Dockerfile.ubuntu.18.04

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2019, Intel Corporation
+#
+# 'recipe' for Docker to build an Debian package
+#
+# Pull base image
+FROM ubuntu:18.04
+Maintainer John.E.Malmberg <John.E.Malmberg@intel.com>
+
+# Install basic tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            autoconf bash curl debhelper dh-make dpkg-dev doxygen gcc \
+            git git-buildpackage locales make patch rpm wget
+
+# libfabric
+ENV UBUNTU_URL="https://launchpad.net/ubuntu/+archive/primary/+files"
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            libibverbs-dev libpsm-infinipath1-dev librdmacm-dev
+
+ARG CACHEBUST=1
```

Comment:
Why is the `CACHEBUST` needed here?

Reply:
Since we can not use Jenkins as a DEBIAN repository, the CACHEBUST is needed to pick up the lastest build from the other Jenkins job.

Reply:
Yes, understand the `CACHEBUST` concept (i introduced it).  My question was why is this one needed here in addition to the one on line 46.

Only one `CACHEBUST` is actually needed per Dockerfile since everything after it will always be done on every image build.  As such you want to put it as low in the Dockerfile as possible to reduce churn.

With a `CACHEBUST` here on line 21, you are going to needlessly refetch the following _libpsm2_ packages on every single docker image build and not use the cache.  These _libpsm2_ packages should never change and shouldn't need to be fetched on every image build.